cardano_node_tests/tests/test_cli.py

```diff
@@ -151,6 +156,68 @@ def test_pretty_utxo(
 
         assert utxo_out == expected_out
 
+    @allure.link(helpers.get_vcs_link())
+    @common.SKIPIF_WRONG_ERA
+    @pytest.mark.parametrize("invalid_param", ("tx_hash", "tx_ix"))
+    @hypothesis.given(filter_str=st.text(alphabet=string.ascii_letters, min_size=1))
+    @common.hypothesis_settings()
```

Comment:
You can likely use `@common.hypothesis_settings(max_examples=200)` or even "300". We want test runtime under one minute. Default value is 100 and I assume this test is very fast.
Same for test bellow.